libafl/src/inputs/mod.rs

```diff
@@ -365,6 +360,7 @@ where
 }
 
 /// A converter that converts from `input` to target bytes
+/// This is necessary because you cannot implemeent `HasTargetBytes` for any input defined inside libafl because of orphan's rule
```

Comment:
`orphan rules`

but also, it's needed because some inputs like Nautilus have state, not sure it has anything to do with orphan rule?